### DISCUSSION TITLE
How can i relock the bootloader?

### DISCUSSION THREAD

**Discussion by Matt-Bos** [kitsuned/PotatoNV#51] [Q&A] [answered]
Hi!! 

As the title says, i want to relock the bootloader but idk how, i tried with the same code that potatoNV gave me but, now i can't relock it. Can you help me please?

**Comment by Matt-Bos** [comment]
> Hi!!
> 
> As the title says, i want to relock the bootloader but idk how, i tried with the same code that potatoNV gave me but, now i can't relock it. Can you help me please?

My phone is a Huawei P Smart 2018.

**Comment by mashed-potatoes** [comment, marked as answer]
- Open PotatoNV
- Connect phone in fastboot mode
- Uncheck `Disable FBLOCK`
- Press Start

**Reply by Matt-Bos** [reply]
Thanks u very much bro

**Reply by MaherAr99** [reply]
does relocking factory reset my phone?

**Reply by MaherAr99** [reply]
@Matt-Bos @mashed-potatoes

**Reply by Matt-Bos** [reply]
> @Matt-Bos @mashed-potatoes

Yes bro, it'll ask you to relock the phone and that option will include a factory reset

**Reply by MaherAr99** [reply]
But unlocking didn't factory reset. When u relock is it mandatory to factory reset?

**Reply by Matt-Bos** [reply]
> But unlocking didn't factory reset. When u relock is it mandatory to factory reset?

Are u sure that u unlocked the phone? When u unlock it's mandatory to factory reset, when u relock it same, it's mandatory

**Reply by MaherAr99** [reply]
Yes that's what i remember. Potatonv unlocking doesn't factory reset your phone. @mashed-potatoes maybe he can tell us if I'm wrong.

**Reply by Matt-Bos** [reply]
PotatoNV doesn't unlock bootloader, it only changes and gives you the unlock code. Once u have it, with Adb u must unlock it with fastboot oem unlock [bootcode of PotatoNV]

**Reply by MaherAr99** [reply]
I'm afraid you are wrong. I unlocked my bootloader using potatoNV on Huawei mate 10 lite and then rooted and installed a custom rom. Just followed the instructions and went smoothly.

**Reply by Matt-Bos** [reply]
Oh, then idk, my phone is a Huawei P Smart, not a mate 10. Sameway try to relock it, it will ask u if u want to relock and factory reset or don't relock it and keep with the info on the phone.

**Reply by MaherAr99** [reply]
Okay thank you my friend. I'll update you when i relock and tell u what happens.

**Reply by mashed-potatoes** [reply]
> does relocking factory reset my phone?

no, follow the instructions above.

**Reply by MaherAr99** [reply]
Thank you so much.
